src/rinterface.c

```diff
@@ -7588,21 +7591,23 @@ SEXP R_igraph_get_adjlist(SEXP graph, SEXP pmode) {
   return result;
 }
 
-SEXP R_igraph_get_adjedgelist(SEXP graph, SEXP pmode) {
+SEXP R_igraph_get_adjedgelist(SEXP graph, SEXP pmode, SEXP ploops, SEXP pmultiple) {
```

Comment:
@ntamas I'm not sure if it's a good idea to have an option that removes multi-edges when dealing with edge indices. If we go down this route, we need to think about which of the several parallel edges to keep, and which to discard. There is no such dilemma with adjlists, but it's a problem with inclists.

For this very reason, `igraph_inclist_init()` does not have a `multiple` argument.

I am concerned that if we expose this functionality in R, people will start relying on it, and we'll be stuck with it.  I suggest _not_ exposing it.

Reply:
Good catch, agreed. We still can expose it to R but not export the two new arguments.

Reply:
IMO it's best not to make use of the `multiple` parameter of `igraph_i_incident()` at all in R/igraph, even internally. I believe that adding this parameter was a mistake. It is currently not used (i.e. not set to `false`) anywhere.

Furthermore, it's not a good idea to start using internal functions from C/igraph unless absolutely necessary. There was a good reason to use `igraph_i_neighbors()` as the features we need here are not _yet_ exposed in `igraph_neighbors()` (but they're planned to). But there is no valid reason to start using the `multiple` argument of `igraph_i_incident()`, and create a reliance on it.